README.md

```diff
@@ -196,22 +206,58 @@ make bats
 make: 'bats' is up to date.
 ```
 
+`publish` allows the publication of all images targeted by 'linux' to a registry.
+
+`ci-init` is dedicated to Jenkins infrastructure for initialising docker and isn't required in other contexts.
+
 ### Building and testing on Windows
 
-From a Powershell console, set first the `IMAGE_TYPE` environment variable defining the Windows flavor ("nanoserver"/"windowsservercore") and  version you want to build.
+#### Building all images
 
-For example:
-```
-New-Item -Path env:IMAGE_TYPE -Value "nanoserver-ltsc2019"
+Run `.\build.ps1` to launch the build of the images corresponding to the "windows" target of docker-bake.hcl.
+
+Internally, the first time you'll run this script and if there is no build-windows.yaml file in your repository, it will use a combination of `docker buildx bake` and `yq` to generate a  build-windows.yaml docker compose file containing all Windows image definitions from docker-bake.hcl. Then it will run `docker compose` on this file to build these images.
+
+You can modify this docker compose file as you want, then rerun `.\build.ps1`.
+It won't regenerate the docker compose file from docker-bake.hcl unless you add the `-OverwriteDockerComposeFile` build.ps1 parameter:  `.\build.ps1 -OverwriteDockerComposeFile`.
+
+Note: you can generate this docker compose file from docker-bake.hcl yourself with the following command (require `docker buildx` and `yq`):
+```console
```

Comment:
````suggestion

```console
````

=> In Markdown, it is safer to always surround blocks (code block in this case) by empty lines

Reply:
Do you have any reference about hits?
I never encountered any issue on GitHub about putting the code directly under the triple back ticks.

Reply:
Oh nevermind, I though it was about an empty line below and not above.